distro/pom.xml

```diff
@@ -88,36 +88,6 @@
                 <resource>
                   <directory>${project.build.directory}/ozone</directory>
                   <excludes>
-                    <exclude>distro/configs/**/ampathforms/*.*</exclude>
-                    <exclude>distro/configs/**/ampathformstranslations/*.*</exclude>
-                    <exclude>distro/configs/**/appointmentservicedefinitions/*.*</exclude>
-                    <exclude>distro/configs/**/appointmentservicetypes/*.*</exclude>
-                    <exclude>distro/configs/**/appointmentspecialities/*.*</exclude>
-                    <exclude>distro/configs/**/attributetypes/*.*</exclude>
-                    <exclude>distro/configs/**/autogenerationoptions/*.*</exclude>
-                    <exclude>distro/configs/**/cohorttypes/*.*</exclude>
-                    <exclude>distro/configs/**/conceptclasses/*.*</exclude>
-                    <exclude>distro/configs/**/concepts/*.*</exclude>
-                    <exclude>distro/configs/**/conceptsources/*.*</exclude>
-                    <exclude>distro/configs/**/drugs/*.*</exclude>
-                    <exclude>distro/configs/**/encounterroles/*.*</exclude>
-                    <exclude>distro/configs/**/encountertypes/*.*</exclude>
-                    <exclude>distro/configs/**/fhirconceptsources/*.*</exclude>
-                    <exclude>distro/configs/**/globalproperties/*.*</exclude>
-                    <exclude>distro/configs/**/idgen/*.*</exclude>
-                    <exclude>distro/configs/**/locationtags/*.*</exclude>
-                    <exclude>distro/configs/**/metadatasets/*.*</exclude>
-                    <exclude>distro/configs/**/metadatasharing/*.*</exclude>
-                    <exclude>distro/configs/**/metadatatermmappings/*.*</exclude>
-                    <exclude>distro/configs/**/ocl/*.*</exclude>
-                    <exclude>distro/configs/**/orderfrequencies/*.*</exclude>
-                    <exclude>distro/configs/**/patientidentifiertypes/*.*</exclude>
-                    <exclude>distro/configs/**/privileges/*.*</exclude>
-                    <exclude>distro/configs/**/programs/*.*</exclude>
-                    <exclude>distro/configs/**/programworkflows/*.*</exclude>
-                    <exclude>distro/configs/**/programworkflowstates/*.*</exclude>
-                    <exclude>distro/configs/**/relationshiptypes/*.*</exclude>
-                    <exclude>distro/configs/**/visittypes/*.*</exclude>
```

Comment:
Why are we deleting this?

Reply:
@vasharma05 this is to maintain the inheritance of config. We were re-defining what already comes with OpenMRS and Ozone which resulted in us missing out on updates in metadata making some parts of the system that depend on these new changes to break.